train.py

```diff
@@ -15,7 +15,7 @@
 attribute_names = ['variance_wavelet_transformed_image', 'skewness_wavelet_transformed_image', 'curtosis_wavelet_transformed_image', 'entropy_image', 'class']
 
 #Read csv-file
-data = pd.read_csv('data\data_banknote_authentication.csv', names=attribute_names)
+data = pd.read_csv('data\data_banknote_authentication.csv', names=attribute_names, header=None)
```

Comment:
dein filepath ist falsch. Probier es mal mit Slash / anstatt backslash \

Lokal wird das bei dir vielleicht funktionieren, weil du auf Windows arbeitest, aber Linux systeme funktionieren leider etwas anders mit den Ordnerstrukturen.

Hier gibts eine Lösung, die für alle Betriebssysteme funktioniert: https://medium.com/@ageitgey/python-3-quick-tip-the-easy-way-to-deal-with-file-paths-on-windows-mac-and-linux-11a072b58d5f


Die Error Message steht auch in deiner github action, falls du es nicht gesehen hast: 
![train py error message](https://user-images.githubusercontent.com/12813089/119935468-d336b480-bf87-11eb-95e5-ecf59bc56e68.jpg)